core/flags.go

```diff
@@ -33,7 +33,7 @@ func (f *MultiFlag) Set(value string) error {
 		return fmt.Errorf(
 			"flag value '%v' was not in the format 'key=val'", value)
 	}
-	key, val := strings.Join(pair[0:1], ""), strings.Join(pair[1:2], "")
+	key, val := strings.Join(pair[0:1], ""), strings.Join(pair[1:], "=")
```

Comment:
With fresh eyes the original line here looks questionable to me and I'm not sure what it's supposed to be getting us. Couldn't we do?:

```golang
func (f *MultiFlag) Set(value string) error {
 	if f.Len() == 0 {
 		f.Values = make(map[string]string, 1)
 	}
	pair := strings.SplitN(value, "=", 2)
	if len(pair) < 2 {
		return fmt.Errorf("flag value '%v' was not in the format 'key=val'", value)
	}
	f.Values[pair[0]] = pair[1]
	return nil
}
```

Do we have an example where the original line does something we need? It'd be nice to have a unit test for this.